### PR TITLE
[FIX] base_delivery_carrier_label test decorators

### PR DESCRIPTION
**Pull request by gurneyalex**
Previous fix was incomplete: the class method should only call other class methods

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]

File: base_delivery_carrier_label/tests/carrier_label_case.py [hunk 1]
```diff
@@ -10,75 +10,81 @@ class CarrierLabelCase(TransactionCase):
     to return the carrier you want to test"""
 
     @classmethod
-    def setUpClass(self):
-        super().setUpClass(self)
-        self._create_order_picking()
+    def setUpClass(cls):
+        super().setUpClass(cls)
```
Comment (yankinmax) [inline base_delivery_carrier_label/tests/carrier_label_case.py]: why not `super().setUpClass()`?

**Comment by yankinmax** [comment]
@gurneyalex 
I've used your fix and tried to rewrite tests for `delivery_roulier_chronopost_fr`
As I've written here https://github.com/OCA/delivery-carrier/pull/830#issuecomment-2233782397
finally I think we shouldn't inherit from this test.

**Review by yankinmax — APPROVED** [review]

**Comment by gurneyalex** [comment]
> @gurneyalex I've used your fix and tried to rewrite tests for `delivery_roulier_chronopost_fr` As I've written here [#830 (comment)](https://github.com/OCA/delivery-carrier/pull/830#issuecomment-2233782397) finally I think we shouldn't inherit from this test.

I will still get this merged, to have some correct code in the module.

**Pull request merged**